etl/src/main/java/org/pih/hivmigration/etl/sql/ProgramMigrator.groovy

```diff
@@ -0,0 +1,252 @@
+package org.pih.hivmigration.etl.sql
+
+class ProgramMigrator extends SqlMigrator {
+
+    void migrate() {
+        executeMysql("Create staging table to receive program data", '''
+            create table hivmigration_programs_raw (
+                source_patient_id int PRIMARY KEY,
+                starting_health_center int,
+                health_center int,
+                health_center_transfer_date date,
+                treatment_status varchar(30),
+                treatment_status_date date,
+                art_start_date date,
+                regimen_outcome varchar(20),
+                regimen_outcome_date date,
+                min_visit_date date,
+                max_visit_date date,
+                enrollment_date date,
+                outcome_date date,
+                outcome varchar(255)
+            );
+        ''')
+
+        executeMysql("Create intermediate staging table", '''
+            create table hivmigration_programs (
+                patient_program_id int PRIMARY KEY AUTO_INCREMENT,
+                source_patient_id int,
+                location_id int,
+                enrollment_date date,
+                art_start_date date,
+                outcome_date date,
+                outcome varchar(255)
+            );
+        ''')
+
+        executeMysql("Create outcome lookup table", '''
```

Comment:
The below outcome table seems unnecessary.  These lookup tables are useful for things like location, where many different steps will likely need to look up locations, if they are associated with different domains of data.  But if this is the only place we need to look this up, this is probably unnecessary.  All of this should likely be moved to later in the migration, where we are already doing various updates based on treatment_status to populate outcome.  We can just do it there.  For here, I would simply add "outcome_concept_id" as a column on the hivmigration_programs_raw table and expect to update it there, rather than have a separate lookup table.  See followup comments below.